deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: discord-karma
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: discord-karma
+  template:
+    metadata:
+      labels:
+        app: discord-karma
+    spec:
+      containers:
+      - name: discord-karam
+        # a public image on quay
+        image: quay.io/jjasghar/discord-karma:1.0.0
```

Comment:
I'd prefer not to have this direct reference in the file. Can you use a template instead?

Reply:
By template you mean something like "PUT IMAGE HERE" or something?

Reply:
I'd say default to a local image name e.g. `discord-karma:1.0.0` and a describing that this should be overridden when it's deployed.

Reply:
updated.